src/Execution/Execution.php

```diff
@@ -65,8 +50,102 @@ public function execute(
             return new ExecutionResult(null, [$error]);
         }
 
-        $data = $context->createExecutor()->execute();
+        $data   = $this->createExecutor($context)->execute();
+        $errors = $context->getErrors();
+
+        return new ExecutionResult($data, $errors);
+    }
+
+    /**
+     * @param Schema        $schema
+     * @param DocumentNode  $documentNode
+     * @param               $rootValue
+     * @param               $contextValue
```

Comment:
Probably missing type-hint for $rootValue and $contextValue (mixed).

Reply:
Fixed.